Extensions/xkit_preferences.js

```diff
@@ -77,25 +77,43 @@ XKit.extensions.xkit_preferences = new Object({
 		}
 
 		const react_add_button = async (button) => {
-			const account_label = await XKit.interface.translate("Account");
-			const menu_label = await XKit.interface.translate("Menu");
+			// const account_label = await XKit.interface.translate("Account");
+			// const menu_label = await XKit.interface.translate("Menu");
+
+			await XKit.css_map.getCssMap();
+			const menuContainerSelector = XKit.css_map.keyToCss("menuContainer");
```

Comment:
Fixed by leaving this as `menuContainer` (to match tumblr's CSS naming as closely as possible, and because the selector itself is generic), and then giving a more specific name to the actual place where we use it)